docs/philosophy.md

```diff
@@ -80,4 +80,7 @@ I do not support child pornography and I ardently [argue against](https://www.yo
 
 During a longer discussion I had arguing in favor of age-of-consent laws with Amos Yee, there is a small section where Amos Yee questioned whether or not child pornography could ever be used in a positive manner. I'd been made familiar with some research indicating it was possible that this could be the case [[1](https://bpspsychub.onlinelibrary.wiley.com/doi/abs/10.1111/j.2044-8341.1995.tb01810.x)][[2](https://bmcpsychiatry.biomedcentral.com/articles/10.1186/1471-244X-9-43)][[3](https://link.springer.com/article/10.1023/B:ASEB.0000029071.89455.53)][[4](https://link.springer.com/article/10.1007%2Fs10508-010-9696-y)][[5](https://www.sciencedirect.com/science/article/abs/pii/S0160252798000351?via%3Dihub)][[6](https://link.springer.com/article/10.1007%2FBF01540933?LI=true)][[7](https://www.sciencedirect.com/science/article/abs/pii/S1359178909000445)], though there would obviously be significant hurdles doing this in an ethical manner. It would be essential to ensure that no new pornography abusing minors would be created and that anything being used as part of any therapy was obtained in an ethical manner, though it's hard to imagine how this would even be possible.
 
-In entertaining this scenario, many who argue against me online are quick to claim that I "advocate for ethical child pornography" or some other reductive statement, though none of these claims are true.
\ No newline at end of file
+In entertaining this scenario, many who argue against me online are quick to claim that I "advocate for ethical child pornography" or some other reductive statement, though none of these claims are true.
+
+<!-- this script injects the googleSchema.js data into philosophy, It uses the ID called in googleSchema.js -->
+<script id='philosophy-faq' type='application/ld+json' dangerouslySetInnerHTML={{ __html:``}} />
```

Comment:
To me the whole idea of adding scripting in a site that is made to work simply in markdown, is not such a great idea since the idea is that destiny can easily edit the content of the pages. By adding this script you are forcing Destiny to learn something that is outside of what he is currently working with.

For the main content creator of this site this seems like a bad idea.